conduct.md

```diff
@@ -120,15 +120,15 @@ TODO: Write an introductory paragraph.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Openness. It is important to be vulnerable in your group so that they understand your strengths and weaknesses within coding. Being open can also allow your team members to help you with problems with your code that you did not catch or understand.
+- Retaining team members ideas. This is a form of respecting everyone's thoughts and opinions. This means to keep or reserve someones idea to let your teammates know that they are heard within the group and their opinions matter.
+- Humbleness. Being humble is a form of humility it is important for everyone in a team to understand that it is not about one person but that it is about the team.
```

Comment:
I really like points 2 and 3, they explain how we can prevent conflicts from starting altogether. But I'd like to see some elaboration or changes in point 1. There is no doubt that openness can help while working in a team, but how does it help specifically in the context of preventing conflicts **between group members**. This section is not really concerned with technical skills while coding.